<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrationWindow</class>
 <widget class="QWidget" name="registrationWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>857</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(238, 174, 202);
background: linear-gradient(90deg, rgba(238, 174, 202, 1) 0%, rgba(183, 177, 216, 1) 100%);
</string>
  </property>
  <widget class="QFrame" name="imgFrame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>353</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>353</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>images/loginImg.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="animationFrame">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>381</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="registrationFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-radius: 10px;
QLabel {
	border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="registrationLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="surnameRegLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите свою фамилию</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameRegLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите своё полное имя</string>
     </property>
    </widget>
    <widget class="QLabel" name="patronymicRefLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string>Введите своё отчество</string>
     </property>
    </widget>
    <widget class="QLabel" name="passportDetailsRegLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите свои паспортные данные</string>
     </property>
    </widget>
    <widget class="QLabel" name="ageRegLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите ваше полное количество лет</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameRegEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="surnameRegEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="patronymicRegEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passportDetailsRegEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ageRegEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="registrationRegButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">:hover {
	background-color: rgb(204, 102, 255);
}</string>
     </property>
     <property name="text">
      <string>Зарегестрироваться</string>
     </property>
    </widget>
    <widget class="QPushButton" name="comeBackButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>420</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">:hover {
	background-color: rgb(204, 102, 255);
}</string>
     </property>
     <property name="text">
      <string>Уже есть аккаунт</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabelReg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="loginFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>381</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="loginlabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string>Вход</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="userNameEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="userNameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите ваше имя</string>
     </property>
    </widget>
    <widget class="QLabel" name="userSurnameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите вашу фамилию</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userSurnameEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="passporDetailsLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Введите ваши паспортные данные</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passportDetailsEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="placeholderText">
      <string>Вводите серию и номер паспорта через пробел</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rememberMeBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Запомнить меня</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="registrationButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">:hover {
	background-color: rgb(204, 102, 255);
}</string>
     </property>
     <property name="text">
      <string>Перейти к регистрации</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
